flink-1.17-connector-bigquery/flink-connector-bigquery/src/main/java/com/google/cloud/flink/bigquery/sink/serializer/AvroToProtoSerializer.java

```diff
@@ -107,6 +107,7 @@ public static DynamicMessage getDynamicMessageFromGenericRecord(
         Schema recordSchema = element.getSchema();
         DynamicMessage.Builder builder = DynamicMessage.newBuilder(descriptor);
         // Get a record's field schema and find the field descriptor for each field one by one.
+        // TODO: Allow Avro schema to be of other named types apart from RECORD: ENUM and FIXED
```

Comment:
Let's create an internal bug for this and remove the TODO from code.

Reply:
Removed the TODO. As per offline discussion, we will only be supporting RECORD type root avro for now.